sysutils/diffutils/Portfile

```diff
@@ -51,20 +50,26 @@ use_xz              yes
 #   die (EXIT_TROUBLE, 0, "%s", _("write failed"));
 #   ^
 #------------------------------------------------------------------------------
-compiler.blacklist-append \
-                    { clang < 900 }
+compiler.blacklist-append   { clang < 900 }
 
-depends_build-append \
-                    port:gettext
+depends_build-append        port:gettext
 
-depends_lib-append \
-                    port:gettext-runtime
+depends_lib-append          port:gettext-runtime
 
-patchfiles          patch-src_system.h.diff
-configure.args      --program-prefix=g \
-                    --infodir=${prefix}/share/info \
-                    --mandir=${prefix}/share/man
+patchfiles                  patch-src_system.h.diff
 
+# Bypass year 2038 awareness code that causes the universal variant to fail.
+pre-configure {
```

Comment:
There's no need to do this in pre-configure. As Ryan pointed out, the flag can be used unconditionally, but even if it should be used only when building universal, the appropriate check is:
```
if {$universal_possible && [variant_isset universal]} {
…
}
```

Reply:
Thanks for the feedback.  I was not sure how this additional new flag would affect 64-bit builds.  That was the main reason I attempted to limit it to universal builds.  And thank you for the code.  I will remember it next time I ever need to limit my code to universal builds.